governance.md

```diff
@@ -0,0 +1,55 @@
+# CNAB Governance
+
+The CNAB Foundation is a non-profit foundation under the [Joint Development Foundation](http://www.jointdevelopment.org/), which is the standards arm of the Linux Foundation.
+
+The CNAB Foundation uses a working group model to create standards and reference implementations.
+
+## Working Groups
+
+Working groups are responsible for defining and approving important parts of the CNAB ecosystem. Each working group is responsible for its own timelines, roadmaps, goals, and deliverables. When a working group reaches a significant milestone, that group may announce a Working Group Approval. This indicates that the working group is ready to release a standard.
+
+Currently, there are two working groups under the foundation:
+
+- The CNAB Specification Working Group: This group works on the [CNAB specifications](https://github.com/deislabs/cnab-spec).
+- The Duffle Working Group: This group works on [duffle](https://github.com/deislabs/duffle), which is the reference implementation of CNAB. The same group works on common libraries and other supporting reference implementations for CNAB.
+
+## The Executive Directors
+
+The executive directors have two sets of responsibilities:
+
+1. Conduct the mundane business associated with the foundation.
+2. Ratify _Working Group Approval_ documents, making them Approved Drafts, which are then published as complete artifacts.
+
+The current executive directors of the CNAB Foundation are (in alphabetical order):
```

Comment:
Do we want to put something here about how long terms are and when the current term will end?